test/subevents.js

```diff
@@ -105,10 +105,10 @@ describe('Subscription Events', function() {
         let eventsReceived = 0;
         const want = 5;
 
-        nc.on('unsubscribe', function(sid, subject) {
+        nc.on('unsubscribe', function() {
             eventsReceived++;
         });
-        const sid = nc.subscribe(subj);
+        const sid = nc.subscribe(subj, () => {});
```

Comment:
Seems sometimes you use function and other times the new syntax, Probably should be consistent.

Reply:
need to change all the other uses before arrow was available to us. Yes it would simplify somethings, but also there are some differences.

Reply:
Do you agree we should try to be consistent and use one or another and not mix and match?